src/ip/Address.h

```diff
@@ -211,6 +212,7 @@ class Address
      \return pointer to buffer received.
      */
     char* toStr(char *buf, const unsigned int blen, int force = AF_UNSPEC) const;
+    SBuf toStrAsSBuf(int force = AF_UNSPEC) const;
```

Comment:
IMO the `AsSBuf` portion of the naming is more harm than good. We should just be deprecating the `char*` API and otherwise keeping the naming the same.
```suggestion
    SBuf toStr(int force = AF_UNSPEC) const;
    /// Deprecated. Use the SBuf method instead.
    char* toStr(char *buf, const unsigned int blen, int force = AF_UNSPEC) const;
```

Reply:
I do agree that toStrAsSBuf() is a non-starter as far as spelling is concerned. However, FWIW, following the above renaming [suggestion](https://github.com/squid-cache/squid/pull/1739#discussion_r1526454769) will _not_ address my primary [concerns](https://github.com/squid-cache/squid/pull/1739#pullrequestreview-1939010545) and, hence, will not move this PR forward.

Reply:
It is now gone

Reply:
> It is now gone

Still here and still used. I cannot provide requested feedback until this is gone.